include/swift/AST/Types.h

```diff
@@ -115,44 +116,44 @@ class RecursiveTypeProperties {
   /// the correct default value and bitwise-or correctly merges things.
   enum Property : unsigned {
     /// This type expression contains a TypeVariableType.
-    HasTypeVariable      = 0x01,
+    HasTypeVariable = 0x01,
 
     /// This type expression contains a context-dependent archetype, either a
     /// PrimaryArchetypeType or OpenedArchetypeType.
-    HasArchetype         = 0x02,
+    HasArchetype = 0x02,
 
     /// This type expression contains a GenericTypeParamType.
-    HasTypeParameter     = 0x04,
+    HasTypeParameter = 0x04,
 
     /// This type expression contains an UnresolvedType.
-    HasUnresolvedType    = 0x08,
-    
+    HasUnresolvedType = 0x08,
+
     /// Whether this type expression contains an unbound generic type.
-    HasUnboundGeneric    = 0x10,
+    HasUnboundGeneric = 0x10,
 
     /// This type expression contains an LValueType other than as a
     /// function input, and can be loaded to convert to an rvalue.
-    IsLValue             = 0x20,
+    IsLValue = 0x20,
 
     /// This type expression contains an opened existential ArchetypeType.
     HasOpenedExistential = 0x40,
 
     /// This type expression contains a DynamicSelf type.
-    HasDynamicSelf       = 0x80,
+    HasDynamicSelf = 0x80,
 
     /// This type contains an Error type.
-    HasError             = 0x100,
+    HasError = 0x100,
 
     /// This type contains a DependentMemberType.
-    HasDependentMember   = 0x200,
```

Comment:
Just a note - making unrelated whitespace changes like this sometimes complicates the review since it's easy to miss value changes...

Reply:
Ah, that must have been `clang-format`, I didn't make any intentional whitespace changes here. I'll revert!